src/main/java/org/apache/commons/io/FileSystem.java

```diff
@@ -73,6 +73,8 @@ public enum FileSystem {
      *
      * @see <a href="https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file">Naming Conventions
      *      (microsoft.com)</a>
+     * @see <a href="https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-createfilea#consoles">
```

Comment:
Is this link useful or is it too technical? The problem is that the above linked "Naming Files, Paths, and Namespaces" does not mention `CONIN$` and `CONOUT$`.

Reply:
I think it is good. Since it is decent technical ref.